controllers/users.js

```diff
@@ -622,6 +632,51 @@ class UserController {
       isDeactivated: updatedAccount.dataValues.isDeactivated
     });
   }
+
+  /**
+   * @param {Object} req - Request from user
+   * @param {Object} res - Response to the user
+   * @returns {Object} Response
+   */
+  async fetchAdminList(req, res) {
```

Comment:
These endpoints haven't been added to swagger documentaion

Reply:
Thanks, I worked on this